tools/standard-test/integtest-runner.al2.dockerfile

```diff
@@ -0,0 +1,94 @@
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This is a docker image specifically for standardize the test environment
+# for both developers and ci/cd tools in OpenSearch
+
+
+FROM amazonlinux:2
+
+# Replace shell with bash so we can source files
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# Add AdoptOpenJDK Repo
+RUN echo -e "[AdoptOpenJDK]\nname=AdoptOpenJDK\nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/7/\$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public" > /etc/yum.repos.d/adoptopenjdk.repo
+
+# Add normal dependencies
+RUN yum update -y && \
+    yum install -y adoptopenjdk-14-hotspot which curl python git tar net-tools procps-ng
+
+# Add Dashboards dependencies
+RUN yum install -y xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib
+
+# Add Notebook dependencies
+RUN yum install -y libnss3.so xorg-x11-fonts-100dpi xorg-x11-fonts-75dpi xorg-x11-utils xorg-x11-fonts-cyrillic xorg-x11-fonts-Type1 xorg-x11-fonts-misc fontconfig freetype && yum clean all
+
+# Add Yarn dependencies
+RUN yum groupinstall -y "Development Tools"
+
+# Remove yum cache
+RUN yum clean all && rm -rf /var/cache/yum/*
+
+# Setup Shared Memory
+RUN  chmod -R 777 /dev/shm
+
+# Must have values for ulimits on the server
+# sudo sysctl -w vm.max_map_count=262144
+# ulimit -n 65535
+# Docker /etc/sysconfig/docker has to have nofiles to 65535
+# OPTIONS="--default-ulimit nofile=65535:65535"
+
+# Create user group
+RUN groupadd -g 1000 opensearch && \
+    useradd -u 1000 -g 1000 -d /usr/share/opensearch opensearch && \
+    mkdir -p /usr/share/opensearch && \
+    chown -R 1000:1000 /usr/share/opensearch
+
+# Environment variables JAVA
+RUN echo "export JAVA_HOME=`dirname $(dirname $(readlink -f $(which javac)))`" >> /etc/profile.d/java_home.sh
+
+# Change User
+USER 1000
+WORKDIR /usr/share/opensearch
+
+# Hard code node version and yarn version for now
+# nvm environment variables
+ENV NVM_DIR /usr/share/opensearch/.nvm
+ENV NODE_VERSION 10.23.1
+# install nvm
+# https://github.com/creationix/nvm#install-script
+RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
```

Comment:
Is there really no better way to install nvm than to download a random script from the internet and run it as root?

Reply:
At the very least we could put this script into this repository so we know it isn't changing

Reply:
This is the best way to install this.
I will save this script to local for now.

Reply:
```
 => [12/19] RUN bash nvm_install.sh                                                                                                                                                                                                                                                                                    3.4s
```